### PR TITLE
fix: Precision

### PR DESCRIPTION
**Pull request by stanleyjones**

### CODE REVIEW

File: client/src/pages/transaction.tsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ export function Transaction() {
       From: <Code>{data.argument.from}</Code>,
       To: <Code>{data.argument.to}</Code>,
       Amount: token
-        ? `${(data.argument.amount / token.precision).toLocaleString()} ${
+        ? `${(data.argument.amount / 10 ** token.precision).toLocaleString()} ${
```
Comment (MavenRain) [inline client/src/pages/transaction.tsx:67]: I get the math, but that 10 looks magical to me . . . from where does it come?

Comment (stanleyjones) [reply client/src/pages/transaction.tsx:67]: We store `amount` and `precision` as integers (u64 maybe?) so to send 1 MFX you'd send an amount of 1,000,000,000 knowing that its precision is 9.

Comment (stanleyjones) [reply client/src/pages/transaction.tsx:67]: Wouldn't the base always be 10? Having trouble imagining otherwise.

Comment (MavenRain) [reply client/src/pages/transaction.tsx:67]: No, you're right, it would be base 10

**Pull request merged**